config/example/node.env

```diff
@@ -90,6 +90,7 @@ NEOFS_OBJECT_PUT_POOL_SIZE_REMOTE=100
 # Storage engine section
 NEOFS_STORAGE_SHARD_POOL_SIZE=15
 NEOFS_STORAGE_SHARD_RO_ERROR_THRESHOLD=100
+NEOFS_STORAGE_IGNORE_MISSING_SHARDS=false
```

Comment:
there is a test for this https://github.com/nspcc-dev/neofs-node/blob/2471-option-to-ignore-uninited-shards/cmd/neofs-node/config/engine/config_test.go. `true` example will be more covering

Reply:
Make `true` for all examples.

Reply:
assert still needs to be added into the test